modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -88,19 +119,28 @@ const onSubmitCellForm = (editor: Editor, cells: HTMLTableCellElement[], api) =>
   });
 };
 
+const getData = (editor: Editor, startCell: SugarElement<Element>, cells: SugarElement<HTMLTableCellElement>[]) => {
+  const selectedCellsOtional = getSelectedCells(cells);
```

Comment:
Small thing - typo, should be `Optional` at the end of the var name

Reply:
We also normally just shorten this to `Opt`